docs/src/man/hts-files.md

```diff
@@ -76,7 +76,7 @@ In SAM/BAM, any description of a reference sequence is stored in the header, und
 ```jlcon
 julia> reader = open(SAM.Reader, "data.sam");
 
-julia> find(header(reader), "SQ")
+julia> findall(SAM.header(reader), "SQ")
```

Comment:
Did you double check that this actually works? :sweat_smile: 

At some point we should probably add doctests...